app/src/main/java/ingsw/group1/findmyphone/Constants.java

```diff
@@ -4,8 +4,7 @@
  * Contains the constants that need to be shared between activities of the application,
  * all of them are defined package-private
  */
-class Constants {
-    final int request = 0, response = 1;
-    final String receivedStringMessage = "receivedStringMessage";
-    final String receivedStringAddress = "receivedStringAddress";
+public final  class Constants {
```

Comment:
For constants capital snake case is usually regarded as better, i.e.:
`RECEIVED_STRING_MESSAGE`